### PR TITLE
Create assignment.hi.md

### PR DESCRIPTION
**Pull request by anurag01a**

### CODE REVIEW

**Comment by jlooper** [comment]
is this ok to merge?

**Comment by Anurag-0-1-A** [comment]
Review required.
@AdityaGarg00 , @innovation-platform , @EliteWarrior315 , Kindly review.

**Comment by ArnoldJohn0495** [comment]
Yes[]()

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# रिसर्च सेंसर और एक्चुएटर्स
+
+## निर्देश
+
+इस पाठ में सेंसर और एक्चुएटर्स शामिल थे। अनुसंधान और एक सेंसर और एक एक्चुएटर का वर्णन करें जिसका उपयोग IoT देव किट के साथ किया जा सकता है, जिसमें शामिल हैं:
```
Comment (AdityaGarg00) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md:5]: I think this (anusandhan karen aur sensors tatha actuactors ka .....) will be much better or something else as it seems little confusing

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* यह क्या करता है
+* अंदर इस्तेमाल होने वाले इलेक्ट्रॉनिक्स / हार्डवेयर
+* क्या यह एनालॉग या डिजिटल है
+* इनपुट या माप की इकाइयाँ और रेंज क्या है
+
+## रूब्रिक
+
+| मानदंड | अनुकरणीय | पर्याप्त | सुधार की जरूरत |
+| -------- | ------------- | -------- | ----------------- |
+| एक सेंसर का वर्णन करें | ऊपर सूचीबद्ध सभी 4 अनुभागों के विवरण सहित एक सेंसर का वर्णन किया। | एक सेंसर का वर्णन किया, लेकिन ऊपर के केवल 2-3 अनुभाग प्रदान करने में सक्षम था | एक सेंसर का वर्णन किया है, लेकिन ऊपर दिए गए अनुभागों में से केवल 1 प्रदान करने में सक्षम था |
```
Comment (AdityaGarg00) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md:16]: can we change these last line's and the last 2nd column lines as it seems a bit confusing? same in line 17

**Review by AdityaGarg00 — CHANGES_REQUESTED** [review]

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# रिसर्च सेंसर और एक्चुएटर्स
+
+## निर्देश
+
+इस पाठ में सेंसर और एक्चुएटर्स शामिल थे। अनुसंधान और एक सेंसर और एक एक्चुएटर का वर्णन करें जिसका उपयोग IoT देव किट के साथ किया जा सकता है, जिसमें शामिल हैं:
+
+* यह क्या करता है
+* अंदर इस्तेमाल होने वाले इलेक्ट्रॉनिक्स / हार्डवेयर
+* क्या यह एनालॉग या डिजिटल है
+* इनपुट या माप की इकाइयाँ और रेंज क्या है
```
Comment (innovation-platform) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md:10]: I think a simpler word can be used for इकाइयाँ. If you think it's fine then it's ok.

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md (hunk 2 above)
Comment (Anurag-0-1-A) [reply 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md:16]: Kindly explain.

Comment (AdityaGarg00) [reply 1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md:16]: I have explained you in private message

**Review by AdityaGarg00 — APPROVED** [review]

**Review by innovation-platform — APPROVED** [review]

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**